### DISCUSSION TITLE
Timeline for Nuxt 3 Alpha release

### DISCUSSION THREAD

**Discussion by harblaith7** [nuxt/nuxt#18210] [Questions]
Can you guys give us an estimated timeline on when the alpha version will be released?

**Comment by ghost** [comment]
The beta version is currently available: https://v3.nuxtjs.org/

**Reply by harblaith7** [reply]
Do you know when alpha will be released? Also do you know if there would be any changes from beta

**Reply by ghost** [reply]
Ah I see what you are asking. Usually alpha comes before beta, but since beta is already out than I expect the next version will either be the Release Candidate or the production ready version.

I imagine there will be quite a few changes from current beta. There are a lot of things still not implemented that will be soon to be feature-par with Nuxt2
